<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Response</class>
 <widget class="QWidget" name="Response">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Response</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPlainTextEdit" name="text_field"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
